boomonkeyboo/scratch_project
# Ck/login page

**Issue Type**
☐ Bug
☑︎ Feature
☐ Tech Debt

**Description**

Login and Signup page

**Steps to Validate Feature**

Use login and signup window on front end

Previous behavior
N/A

Expected behavior
Follow steps

Screenshots & Videos


Additional context
